packages/api/src/controllers/access-control.ts

```diff
@@ -210,6 +221,16 @@ app.post(
             "Content is gated and requires an access key"
           );
         }
+
+        const [keys] = await db.accessControlKey.find({
```

Comment:
I don't like how we are adding 2 more IOs in the API that is already slow due to db access. Do we need to fetch the access key here? Didn't we already fetch it somewhere in this handler to validate the jwt?